client/.eslintrc.json

```diff
@@ -14,7 +14,6 @@
 	"parser": "babel-eslint",
 	"rules": {
 		"react/jsx-filename-extension": 0,
-		"react/prop-types": 0,
```

Comment:
Should set this to 2 instead of removing it.  Refer to [this](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prop-types.md) as to why.

Reply:
We extend from the airbnb eslint settings so should this should be on automatically if we remove it so we probably can still remove this but I maybe wrong.